### ISSUE TITLE
Normalize otherId name=label (content metadata)

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss/dor-services-app#3245]
For example, druid:bd251xz8088:

```
<identityMetadata>
  <otherId name="label">4084148</otherId>
</identityMetadata>
```
This should be removed in normalization.

**Comment by peetucket** [comment]
Initial action for ticket: 
* write a report to find any items which other <otherId name="label"> in identityMetadata datastream and provide to Andrew

**Comment by edsu** [comment]
I added a [report script](https://github.com/sul-dlss/dor-services-app/pull/3270) and ran it on sdr-deploy against `/opt/app/deploy/dor-services-app/druids.txt`. It looks like there were 308 otherId labels. I've attached the CSV here.

@peetucket @andrewjbtw does this help determine what next steps should be?

[identity-otherids_labels.csv](https://github.com/sul-dlss/dor-services-app/files/7973215/identity-otherids_labels.csv)

**Comment by justinlittman** [comment]
@edsu The attached reported is empty -- it contains the headers only.

**Comment by edsu** [comment]
Whoops, wrong CSV. Thanks for catching that @justinlittman. Here's the correct one:

[identity-otherids_labels.csv](https://github.com/sul-dlss/dor-services-app/files/7973735/identity-otherids_labels.csv)

**Comment by justinlittman** [comment]
That doesn't look very interesting. Can we ignore otherId name=label @andrewjbtw ?

**Comment by andrewjbtw** [comment]
These are almost all media items from a small number of collections. I'm asking the media digitization team.

**Comment by andrewjbtw** [comment]
Ok, got it figured out. This can be ignored and left unmapped.

With only a couple of exceptions, the values in the `otherid=label` field are the Symphony catkey for the collection that the druid is a member of. This might be a legacy of different ways SDR used in the past to show object relationships, or a past approach to SearchWorks release. In any case, the collection relationship is now in the RELS-EXT and we don't need the identityMetadata value.

I looked at the four exceptions where the value is either "Unknown" or a druid. I'm confident those labels are not needed.

**Comment by edsu** [comment]
@justinlittman Is there a programmatic way to flag things as ok to drop? Or do we just close this issue?

**Comment by justinlittman** [comment]
Action for this ticket: Remove otherId[@name=label] as part of [identity normalization](https://github.com/sul-dlss/dor-services-app/blob/main/app/services/cocina/normalizers/identity_normalizer.rb).

**Comment by edsu** [comment]
Fixed in #3270
